### ISSUE TITLE
Arduino 1.6 support

### ISSUE THREAD

**Issue by mattpackwood** [hexbright/samples#7]
It appears there is no support for Arduino 1.6......

**Comment by disquisitioner** [comment]
Are you seeing problems compiling Hexbright code on Arduino 1.6, and if so which sketches and what warnings/errors?  Happy to take a look.  I compiled a copy of the original factory code, plus a modified version I use on my hexbright with Arduino 1.6 on my Windows machine and didn't encounter any errors or warnings.

I have seen compile errors using Arduino 1.6 with other pre-existing sketches, e.g., the new compiler now requires PROGMEM allocations be declared const but that's not an issue for any Hexbright code I've come across.

**Comment by mattpackwood** [comment]
My problem is more basic. I can't add the boards.txt file to Mac 1.6

**Comment by disquisitioner** [comment]
I was able to add the Hexbright information to the boards.txt file on my Mac running 1.6.  Here's what I did:

*) Make sure the Arduino IDE is not running as you will be modifying its configuration files.
*) Launch Finder, navigate to the Applications folder, right click on 'Arduino', and select 'Show Package Contents' from the pop-up convenience menu.
*) Within the 'Contents' folder, navigate down to ./Resources/Java/hardware/arduino/avr where you'll see 'boards.txt'.  Right mouse click on boards.txt and open with your favorite text editor.
*) Paste the Hexbright board info at the end of boards.txt, save the file, and exit the editor
*) Restart the Arduino 1.6 IDE and confirm that Hexbright now shows at the end of the list of 'Arduino AVR boards' in the Tools -> Board submenu.

This is the same procedure used for installing on Arduino 1.0.X so perhaps the problem you're having is different.   I did test compile my Hexbright sketch with Hexbright as the target board and got no errors, but I don't have my Hexbright here so can't test uploading the compiled sketch just yet...
